### 上位机可修改参数

# 雷达型号选择: N10, N10_P
lidar_name: N10

# 定义 base_link -> laser_link 的固定 TF 变换，单位cm和度
x: 0
y: 0
z: 0
roll: 0
pitch: 0
yaw: 0

# SLAM_TOOLBOX建图or定位模式，mapping/localization
mode: mapping

# 是否使能飞控输出坐标
px4_flag: 1

# 是否使能串口输出坐标
uart_flag: 1

### 上位机不可修改参数

# 雷达驱动发布的话题
scan_topic: /scan_raw

# 雷达串口设备
serial_port_: /dev/usb_robot
